src/tls/cockpit-certificate-helper.in

```diff
@@ -114,6 +173,9 @@ main() {
 
     # Dispatch subcommand
     case "$1" in
+        selfsign)
+            cmd_selfsign
```

Comment:
Let's please move this commit into a future PR that actually uses this new verb. Dead code is never nice..

Reply:
Ya.  I was thinking that's a good idea.  And rename this PR, then.